bin/gwu/gwuLib.ml

```diff
@@ -11,6 +11,7 @@ let out_dir = ref ""
 let raw_output = ref false
 let sep_limit = ref 21
 let separate_list = ref []
+let dummy = ref [ "" ]
```

Comment:
Why do you use a reference here? This variable is only used once and never mutated.

Reply:
the ext-files list if of the format `(string * string list ref) list`
I need it in
```
    let ext_files =
      List.fold_left (fun acc f -> (f, dummy) :: acc) gen.ext_files ext_files_2
    in

```